webapp/src/pages/PerturbationTestingSummary.tsx

```diff
@@ -10,7 +10,7 @@ import { isPipelineSelected } from "utils/helpers";
 
 export const behavioralTestingDescription = (
   <Description
-    text="Assess if your model is robust to small modifications."
+    text="Assess your pipeline's linguistic capabilities with Robustness Invariance tests."
```

Comment:
Not a huge difference, but I think "tests" sounds more like we want the user to make tests, whereas to me "testing" sounds more like we have done the testing and the user does his analysis using the result. I don't know why, though, so I might be wrong.
```suggestion
    text="Assess your pipeline's linguistic capabilities with Robustness Invariance testing."
```

Reply:
I think I still prefer tests, but let's see what Lindsay thinks.

Reply:
I think I agree with Gab here. I think it's clear that we are presenting Robustness Invariance tests, and the user can use them to do analysis/assessment. Also, we're not showing the "testing" process. However, along that line of thought, one thing you could consider that might address @JosephMarinier's concern is to add "results" (since what we are showing are the results of the tests), so something like this: 

```suggestion
    text="Assess your pipeline's linguistic capabilities with Robustness Invariance test results."
```
or
```suggestion
    text="Assess your pipeline's linguistic capabilities with the results of Robustness Invariance tests."
```

And/orr invert the order of the sentence, like this:

```suggestion
    text="Use Robustness Invariance test results to assess your pipeline's linguistic capabilities."
```

But I kind of like that the way you wrote it starts with the assessment purpose to put it directly in the user's mind, so I wouldn't necessarily pick the last suggestion. And I don't feel super strongly about keeping it as is vs. adding "results" - it might make it more awkward.

Reply:
I would leave it as is personnaly.